include-media.js

```diff
@@ -30,6 +30,9 @@
         breakpoints = JSON.parse(removeQuotes(data));
       } catch (err) {}
     }
+    else {
```

Comment:
Do you mind putting the `else` on the previous line, next to the closing curly brace, as per the rest of the file?

Also, should it return `null` instead of `false`?